tests/scss/scss/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4678,6 +4784,40 @@ label {
     )
   );
 }
+@forward "src/list";
+@forward "src/list" as list-*;
+@forward "src/list" as list-*;
+@forward "src/list" hide list-reset, $horizontal-list-gap;
+@forward "src/list" hide list-reset, $horizontal-list-gap,
+  $horizontal-list-gap-foo, $horizontal-list-gap-bar;
+
+@use "src/code";
+@use "src/code" as c;
+@use "src/code" as c;
+@use "src/code" as *;
+@use "library" with ($black: #222, $border-radius: 0.1rem);
+@use "library" with
+  ($black: #222, $border-radius: 0.1rem, $left: 10px, $right: 400px);
+@use "library" with
+  (
+    $black: #222,
+    $border-radius: 0.1rem,
+    $left: 10px,
+    $right: 400px,
+    $box-shadow: 0 0.5rem 1rem rgba($black, 0.15)
+  );
+@use "veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLong"
+  with
+  (
+    $black: #222,
+    $border-radius: 0.1rem,
+    $left: 10px,
+    $right: 400px,
+    $box-shadow: 0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15),
+    0 0.5rem 1rem rgba($black, 0.15)
+  );
```

Comment:
Do you think we should move `with` on the next line?

Reply:
Yes I am just not sure how. Still learning the ropes here.